app/src/main/java/com/amazonaws/kinesisvideo/demoapp/activity/WebRtcActivity.java

```diff
@@ -185,7 +187,7 @@ private void initWsConnection() {
         final String masterEndpoint = mWssEndpoint + "?" + Constants.CHANNEL_ARN_QUERY_PARAM + "=" + mChannelArn;
 
         // See https://docs.aws.amazon.com/kinesisvideostreams-webrtc-dg/latest/devguide/kvswebrtc-websocket-apis-1.html
```

Comment:
https://docs.aws.amazon.com/kinesisvideostreams-webrtc-dg/latest/devguide/kvswebrtc-websocket-apis.html

Reply:
I linked ConnectAsViewer above the viewer URL and ConnectAsMaster above the master URL